libs/providers/flagd/src/lib/service/in-process/data-fetch.ts

```diff
@@ -4,7 +4,7 @@
 export interface DataFetch {
   connect(
     dataFillCallback: (flags: string) => void,
-    connectCallback: () => void,
+    reconnectCallback: () => void,
```

Comment:
@Kavindu-Dodan I've specifically called this a "reconnectCallback" because it's called only on connections AFTER the initial one (see other comments).